File: opcua-webservice-spring/core/src/main/java/org/opcfoundation/webserver/types/WriteVariableValue.java
package org.opcfoundation.webserver.types;

import org.eclipse.milo.opcua.stack.core.types.builtin.DataValue;

public class WriteVariableValue {
    private final UaChildId variableId;
    private final DataValue value;

    public WriteVariableValue(
            UaChildId variableId,
            DataValue value)
    {
        this.variableId = variableId;
        this.value = value;
    }

    public UaChildId getVariableId() {
        return variableId;
    }

    public DataValue getValue() {
        return value;
    }
}

File: opcua-webservice-node/src/server/types/common/WriteVariableValue.ts
import { DataValue } from "opcua-webapi";
import { UaChildId } from "./UaChildId";
import { UaDataValue } from "opcua-webapi-ts";

export class WriteVariableValue {
    private readonly _variableId: UaChildId;
    private readonly _value: UaDataValue;

    constructor(variableId: UaChildId, value: UaDataValue) {
        this._variableId = variableId;
        this._value = value;
    }

    get variableId(): UaChildId {
        return this._variableId;
    }

    get value(): UaDataValue {
        return this._value;
    }
}